filenames.gypi

```diff
@@ -103,6 +103,7 @@
       'atom/app/node_main.h',
       'atom/app/uv_task_runner.cc',
       'atom/app/uv_task_runner.h',
+      'atom/browser/api/atom_api_app_mac.mm',
```

Comment:
This file should be included only if `mas_build==1`, check the conditional includes at the very end of the file.